File: python/Q_283.py
class Solution:
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: void Do not return anything, modify nums in-place instead.
        """
        j=0
        i=0
        while i<len(nums):
            if nums[i]!=0:
                nums[j]=nums[i]
                j=j+1
            i=i+1
        while j<len(nums):
            nums[j]=0
            j=j+1

File: Q_283.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int i,j,cap=nums.size();
        for(j=0,i=0;i<cap;){
            while(i<cap&&nums[i]==0)
                i++;
            if(i==cap)
                break;
            nums[j++]=nums[i++];
        }
        while(j<cap)
            nums[j++]=0;
    }
};
